scripts/scrape.py

```
@@ -0,0 +1,53 @@
+    "https://en.wikipedia.org/wiki/World_Solar_Challenge",
+    "https://en.wikipedia.org/wiki/Formula_Sun_Grand_Prix",
+    "https://en.wikipedia.org/wiki/Solar_car",
+    "https://en.wikipedia.org/wiki/Solar_energy",
+]
+
+def scrape():
+    documents = []
+    for i, target in enumerate(target_urls):
+        print("Scraping:", target)
+
+        response = requests.post(
```

Ruff (0.15.13)

[error] Probable use of `requests` call without timeout

(S113)